setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
@marqh @dkillick @pp-mo or is it `scitools-iris` ?

@marqh Have you tested doing a `python setup.py build` and `python setup.py install` etc just to make sure that nothing odd is happening?